build.sbt

```diff
@@ -93,20 +87,16 @@ libraryDependencies ++= Seq(
   "org.mock-server" % "mockserver-netty" % "5.11.1" % Test,
   "com.dimafeng" %% "testcontainers-scala-mysql" % testcontainersScalaV % Test,
   "com.dimafeng" %% "testcontainers-scala-mongodb" % testcontainersScalaV % Test,
-  "org.flywaydb" % "flyway-core" % "7.0.0" % Test,
-  "org.broadinstitute.dsde.workbench" %% "sam-client" % "0.1-9d0baea",
-  "org.broadinstitute.cromwell" %% "cromwell-client" % "0.1-8b413b45f-SNAP"
+  "org.flywaydb" % "flyway-core" % "7.3.2" % Test,
+  "org.broadinstitute.dsde.workbench" % "sam-client_2.12" % "0.1-9d0baea",      // Should become available for 2.13 once a release happens ( https://github.com/broadinstitute/sam/pull/491 )
+  "org.broadinstitute.cromwell" % "cromwell-client_2.12" % "0.1-8b413b45f-SNAP" // Contains only Java, pinning on 2.12
 )
 
 // Flyway may be run with system properties:
 // i.e: sbt -Dflyway.url=jdbc:mysql://DB_HOST:DB_PORT/DB_NAME -Dflyway.user=root -Dflyway.password=abc123
 
 flywayCleanDisabled := true
 
-releaseSettings
-
-shellPrompt := { state => "%s| %s> ".format(GitCommand.prompt.apply(state), version.value) }
```

Comment:
Any comment on why these were removed? They seem "maybe not ideal" but not actively part of updating to 2.13?

Reply:
`releaseSettings`: creating a key without a value appears to be disallowed in the latest SBT. Updating SBT was required to upgrade Scala.

`shellPrompt`: this customization displays the current Git branch on the SBT shell. Nothing wrong with that specifically, except that `GitCommand` comes from [the `sbt-git` plugin](https://github.com/sbt/sbt-git/releases) that appears to be abandonware and was removed in `project/plugins.sbt`